assets/build/install.sh

```diff
@@ -153,7 +153,7 @@ if [[ -d ${GEM_CACHE_DIR} ]]; then
   chown -R ${GITLAB_USER}: ${GITLAB_INSTALL_DIR}/vendor/cache
 fi
 
-exec_as_git bundle install -j"$(nproc)" --deployment --without development test mysql aws
+exec_as_git bundle install -j"$(nproc)" --deployment --without development test
```

Comment:
Not sure about this. Do we still need `mysql` and `aws`? :thinking: 

Refs https://github.com/sameersbn/docker-gitlab/pull/1961/files#r337386468

Reply:
You are right we could also drop `aws`and `mysql` according to https://docs.gitlab.com/ce/install/installation.html#install-gems . Mysql should be though I think it got missed during the rebasing in #1961 .

Reply:
But to have it as a correctly behaviour you should have passed it in the `--without` option so this would install `aws`and `mysql`. So it needs to be like this:

```suggestion
exec_as_git bundle install -j"$(nproc)" --deployment --without development test aws mysql
```

Reply:
I agree :+1: Let's the removal of `aws` and `mysql` in a separate PR :+1:

Reply:
Tracked via https://github.com/sameersbn/docker-gitlab/issues/2035